md/workspaces-and-repositories.md

```diff
@@ -101,11 +101,43 @@ Bonita studio Git integration is based on th EGit eclipse plugin.
 Git command available in Studio interface:
 
 * **Share with Git**  
-This action connect the current repository to Git and share it on a remote.
-To configure the remote see the following [Egit userguide](http://wiki.eclipse.org/EGit/User_Guide#Working_with_remote_Repositories) or the [Share on GitHub howto](share-a-repository-on-github.md).
+This action connects the current repository to Git and shares it on a remote.
+To configure the remote, see the following [Egit userguide](http://wiki.eclipse.org/EGit/User_Guide#Working_with_remote_Repositories) or the [Share on GitHub howto](share-a-repository-on-github.md).
 
 * **Clone**  
-Create an new Studio repository from an existing Git repository (must contains a proper Bonita project). If the remote repository version is lower than the Studio, a migration will be applied on the cloned repository. Be carefull before pushing a migration on the remote. All contributors will have to use the proper Studio version.
+Create a new Studio repository from an existing Git repository (that must contain a proper Bonita project). If the remote repository version is lower than the Studio, a migration will be applied on the cloned repository. Be careful before pushing a migrated repository back to the remote: all contributors will have to use the proper Studio version.
+If you used Bonita on Git before Bonita 7.7.0, chances are high the .gitignore file structure of your project is different from the one created by Bonita Studio, which may cause some issues. Before you clone the repository, manually edit your .gitignore file so it becomes:
```

Comment:
Actually we do not really care about the .gitignore file.
The mandatory one is the .project